backend/src/entries/entries.service.ts

```diff
@@ -17,41 +17,51 @@ export class EntriesService {
         familyNameKana: true,
         personalNameKana: true,
         gender: true,
-        birthDay: true,
+        birthday: true,
         prefecture: true,
         tel: true,
         email: true,
         isAccompanied: true,
-        visitDate: true,
+        visitDay: true,
+        visitTime: true,
       },
     });
   }
 
   async addEntry(entry: AddEntryDto): Promise<EntryDto> {
+    let isAccompaniedBoolean = true;
+    if (entry.isAccompanied === 'なし') {
+      isAccompaniedBoolean = false;
+    }
     const data = {
       familyName: entry.familyName,
       personalName: entry.personalName,
       familyNameKana: entry.familyNameKana,
       personalNameKana: entry.personalNameKana,
       gender: entry.gender,
-      birthDay: new Date(entry.birthDay),
+      birthday: new Date(entry.birthday),
       prefecture: entry.prefecture,
       tel: entry.tel,
       email: entry.email,
-      isAccompanied: entry.isAccompanied,
-      visitDate: new Date(entry.visitDate),
+      isAccompanied: isAccompaniedBoolean,
+      visitDay: new Date(entry.visitDay),
+      visitTime: entry.visitTime,
     };
 
     return this.prisma.entries.create({ data });
   }
 
   async updateEntry(id: number, entry: UpdateEntryDto): Promise<EntryDto> {
+    let isAccompaniedBoolean = true;
+    if (entry.isAccompanied === 'なし') {
+      isAccompaniedBoolean = false;
+    }
```

Comment:
```suggestion
   const isAccompanied = entry.isAccompanied === 'あり'
```

本当は「あり」はPrismaのenumから取得したいですが、これ↑ではダメですか？

Reply:
@kentosity 

下記同様、isAccompaniedはboolean型で取得されるので、不要な記述となりました。
https://github.com/wannabehime/training_simplified_wecall/pull/11#discussion_r1940940342

下記で修正しました。
[5c04e1a](https://github.com/wannabehime/training_simplified_wecall/pull/21/commits/5c04e1ab25eb416dc33517f106f3465a09f7f820)